server/routerlicious/packages/lambdas/src/moira/README.md

```diff
@@ -0,0 +1,5 @@
+Lambda that processes only ops related to PropertyDDS.
+From each op a changeSet extracted and submitted to Moira service.
+
+For each PropertyDDS lambda will create a branch if it doesn't exist.
+Later, all corresponding changeSets will committed to that branch.
```

Comment:
A little more description would be nice. Like how does moira know about the property set DDS changes, the schema of such ops etc.